src/js/config/Config.ts

```diff
@@ -1,8 +1,45 @@
 /* eslint no-redeclare: 0 */
-import ConfigDev from "./Config.dev.js";
-import ConfigTest from "./Config.test.js";
 
-var Config = {
+import ConfigDev from "./Config.dev";
+import ConfigTest from "./Config.test";
+
+interface IConfiguration {
```

Comment:
Uh, didnt we say that we dont want to do it like this? 🤔

Reply:
We still have it in our tslint config